js/views/questionView.js

```diff
@@ -240,6 +246,10 @@ class QuestionView extends ComponentView {
     Adapt.trigger('questionView:showInstructionError', this);
   }
 
+  addErrorClass() {
+    this.model.toggleClass('has-error', true);
+  }
+
```

Comment:
A whole function is not needed for one line.
Can you add it into `showInstructionError`, before the event trigger please?

Reply:
I created a separate function for this because showInstructionError() doesn't actually relate to adding a generic error class to a component. Perhaps showInstructionError() should be renamed showError()?

Reply:
Before the event trigger please. It should be in a fit state when the event is called such that the event handles have the `has-error` class. 
No, please don't change the function name, I think "instruction error" is a good description of the phase of the execution.

Reply:
Done.